### PR TITLE
[NavigationDrawer] Initial doc write-up of overview, and usage.

### LINKED ISSUE #4925: [NavigationDrawer] Write docs
Write initial doc for Navigation Drawer.

### PR DESCRIPTION
**Pull request by yarneo**
This includes an initial write-up of the navigation drawer docs. closes #4925

### CODE REVIEW

File: components/NavigationDrawer/README.md [hunk 1]
```diff
@@ -1,4 +1,149 @@
...
+- [Installation](#installation)
+  - [Installation with CocoaPods](#installation-with-cocoapods)
+  - [Importing](#importing)
+- [Usage](#usage)
+
+- - -
+
+## Overview
+
+Navigation Drawer is currently an [alpha component](https://github.com/material-components/material-components-ios/blob/develop/contributing/alpha_components.md). Therefore, clients that wish to use Navigation Drawer in their app will need to manually clone the repo and add the code to their project.
```
Comment (codeman7) [inline components/NavigationDrawer/README.md:51]: I think we are supposed to wrap lines even in docs.

File: components/NavigationDrawer/README.md [hunk 2]
```diff
@@ -1,4 +1,149 @@
...
+Currently this component is an [alpha component](https://github.com/material-components/material-components-ios/blob/develop/contributing/alpha_components.md) and therefore doesn't support installation via CocoaPods. This is intended. 
+
+### Importing
+
+To import the component:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+import MaterialComponents.MaterialNavigationDrawer
```
Comment (codeman7) [inline components/NavigationDrawer/README.md]: `MaterialComponentsAlpha.MaterialNavigationDrawer`

**Review by codeman7 — APPROVED** [review]

**Review by ianegordon — COMMENTED** [review]
Shouldn't this go in the docs/ subdirectory and get generated by our script?

**Comment by jverkoey** [comment]
+1 please use the doc generator - no need to maintain a table of contents plus you get badges!

**Comment by yarneo** [comment]
ah wonderful!

**Pull request merged**